virttest/bootstrap.py

```diff
@@ -45,7 +45,7 @@
                       'v2v': basic_program_requirements,
                       'libguestfs': basic_program_requirements}
 
-mandatory_headers = {'qemu': ['Python.h', 'types.h', 'socket.h', 'unistd.h'],
+mandatory_headers = {'qemu': ['types.h', 'socket.h', 'unistd.h'],
```

Comment:
i guess this is in relation with python3-devel dependency removal, but if there is a reason, not documented clearly in commit message,.